code/datums/brain_damage/schizo.dm

```diff
@@ -0,0 +1,219 @@
+///////////////BRAINWASHING////////////////////
+
+/datum/brain_trauma/severe/split_personality/brainwashing
+	name = "Split Personality"
+	desc = "Patient's brain is split into two personalities, which randomly switch control of the body."
+	scan_desc = "complete lobe separation"
+	gain_text = ""
+	lose_text = "<span class='notice'>You are free of your brainwashing.</span>"
+	can_gain = FALSE
+	var/codeword
+	var/objective
+
+/datum/brain_trauma/severe/split_personality/brainwashing/New(obj/item/organ/brain/B, _permanent, _codeword, _objective)
+	..()
+	if(_codeword)
+		codeword = _codeword
+	else
+		codeword = pick(strings("ion_laws.json", "ionabstract")\
+			| strings("ion_laws.json", "ionobjects")\
+			| strings("ion_laws.json", "ionadjectives")\
+			| strings("ion_laws.json", "ionthreats")\
+			| strings("ion_laws.json", "ionfood")\
+			| strings("ion_laws.json", "iondrinks"))
+
+/datum/brain_trauma/severe/split_personality/brainwashing/on_gain()
+	..()
+	var/mob/living/mental/split_personality/traitor/traitor_backseat = stranger_backseat
+	traitor_backseat.codeword = codeword
+	traitor_backseat.objective = objective
+
+/datum/brain_trauma/severe/split_personality/brainwashing/make_backseats()
+	stranger_backseat = new /mob/living/mental/split_personality/traitor(owner, src, codeword, objective)
+	owner_backseat = new(owner, src)
+
+
+/datum/brain_trauma/severe/split_personality/brainwashing/get_ghost()
+	set waitfor = FALSE
+	var/datum/ghosttrap/G = get_ghost_trap("split personality")
+	G.request_player(stranger_backseat, "Would you like to play as [owner]'s split personality?", 60 SECONDS)
+	addtimer(CALLBACK(src, .proc/reset_search), 60 SECONDS)
+
+/datum/brain_trauma/severe/split_personality/brainwashing/on_life()
+	return //no random switching
+
+/datum/brain_trauma/severe/split_personality/brainwashing/on_hear(message, speaker, message_language, raw_message, radio_freq)
+	if(owner.disabilities & DEAF || owner == speaker)
+		return message
+	if(findtext(message, codeword))
+		message = replacetext(message, codeword, "<span class='warning'>[codeword]</span>")
+		addtimer(CALLBACK(src, /datum/brain_trauma/severe/split_personality.proc/switch_personalities), 10)
+	return message
+
+/datum/brain_trauma/severe/split_personality/brainwashing/on_say(message)
+	if(findtext(message, codeword))
+		return "" //oh hey did you want to tell people about the secret word to bring you back?
+	return message
+
+/mob/living/mental/split_personality/traitor
+	name = "split personality"
+	real_name = "unknown conscience"
+	var/objective
+	var/codeword
+
+/mob/living/mental/split_personality/traitor/Login()
+	..()
+	to_chat(src, "<span class='notice'>As a brainwashed personality, you cannot do anything yet but observe. However, you may gain control of your body if you hear the special codeword, switching places with the current personality.</span>")
+	to_chat(src, "<span class='notice'>Your activation codeword is: <b>[codeword]</b></span>")
+	if(objective)
```

Comment:
I'm pretty sure we don't have traitor/antag objectives on Aurora.

Reply:
These aren't traitor/antag objectives, these are brainwash objectives.